content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
+This enables a strong [defense in depth](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) strategy that can be used to further strengthen the security of your mesh.
+
+For example, you may choose to only allow traffic to port `9080` of our `reviews` application.
+In the event of a compromised pod or security vulnerability in the cluster, this may limit or stop an attackers progress.
+
+### Securing egress traffic
+
+A common misconception is that options like [`outboundTrafficPolicy: REGISTRY_ONLY`](/docs/tasks/traffic-management/egress/egress-control/#envoy-passthrough-to-external-services) acts as a security policy preventing all access to undeclared services.
+However, this is not a strong security boundary as mentioned above, and should be considered best-effort.
+
+While this is useful to prevent accidental dependencies, if you want to secure egress traffic, and enforce all outbound traffic goes through a proxy, you should instead rely on an [Egress Gateway](/docs/tasks/traffic-management/egress/egress-gateway/).
+When combined with a [Network Policy](/docs/tasks/traffic-management/egress/egress-gateway/#apply-kubernetes-network-policies), you can enforce all traffic, or some subset, goes through the egress gateway.
+This ensures that even if a client accidentally or maliciously bypasses their sidecar, the request will be blocked.
+
+## Configure TLS verification in Destination Rule when using TLS origination
+
+Istio offers the ability to [originate TLS](/docs/tasks/traffic-management/egress/egress-tls-origination/) from the sidecar proxy.
+This enables applications that send plaintext HTTP traffic to be transparently "upgraded" to HTTPS.
+
+Care must be taken when configuring the `DestinationRule`'s `tls` setting to specify the `caCertificates` field.
+When this is not set, the servers certificate will not be verified.
+
+For example:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: google-tls
+spec:
+  host: google.com
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      caCertificates: /etc/ssl/certs/ca-certificates.crt
+{{< /text >}}
+
+## Gateways
+
+When running an Istio [gateway](/docs/tasks/traffic-management/ingress/), there are a few resources involved:
+
+* `Gateway`s, which controls the ports and TLS settings for the gateway.
+* `VirtualService`s, which control the routing logic. These are associated with `Gateway`s by direct reference in the `gateways` field and a mutual agreement on the `hosts` field in the `Gateway` and `VirtualService`.
+
+### Restrict `Gateway` creation privileges
+
+It is recommended to restrict creation of Gateway resources to trusted cluster administrators. This can be achieved by [Kubernetes RBAC policies](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) or tools like [Open Policy Agent](https://www.openpolicyagent.org/).
```

Comment:
I may be misunderstanding, but doesn't this line already recommend OPA?

Reply:
I mean that it mentions OPA as a way to restrict Gateway creation to administrators, but it might be worth saying that OPA could be used for more of these recommendations too

Reply:
Ah, got it. Agreed :+1: